org/ystia/topologies/a4c_yorc_basic/types.yml

```diff
@@ -14,15 +14,15 @@ imports:
 - org.alien4cloud.alien4cloud.webapp:2.1.0
 - org.alien4cloud.java.jdk.linux:2.1.0
 
-- org.ystia.terraform.linux.terraform:2.1.0-SNAPSHOT
-- org.ystia.yorc.alien4cloud:2.1.0-SNAPSHOT
-- org.ystia.ansible.linux.ansible:2.1.0-SNAPSHOT
-- org.ystia.terraform.pub:2.1.0-SNAPSHOT
-- org.ystia.yorc.linux.ansible:2.1.0-SNAPSHOT
-- org.ystia.yorc.experimental.consul.linux.ansible:2.1.0-SNAPSHOT
-- org.ystia.yorc.pub:2.1.0-SNAPSHOT
-- org.ystia.ansible.pub:2.1.0-SNAPSHOT
-- org.ystia.yorc.experimental.consul.pub:2.1.0-SNAPSHOT
+- org.ystia.terraform.linux.terraform:2.2.0-SNAPSHOT
```

Comment:
Could indent all these imports, adding 2 spaces, looks like this is why the release script didn't catch then

Reply:
And the 2 others are not detected as they are not named `types.yaml`